orchestrators/FinanceOperations/src/Coordinators/GLPostingCoordinator.php

```
@@ -108,11 +111,13 @@ public function postToGL(GLPostingRequest $request): GLPostingResult
             // Validate account mappings
-            $mappingResult = $this->accountMappingRule->check((object)[
-                'tenantId' => $request->tenantId,
-                'subledgerType' => $request->subledgerType,
-                'transactionTypes' => $request->options['transaction_types'] ?? [],
-            ]);
+            $mappingResult = $this->accountMappingRule->check(
+                RuleContext::forGlAccountMappingValidation(
+                    tenantId: $request->tenantId,
+                    subledgerType: $request->subledgerType,
+                    transactionTypes: $request->options['transaction_types'] ?? [],
+                )
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Nexus\FinanceOperations\DTOs\RuleContext' in method 'postToGL'. (undefined)

(StaticAccess)